.github/workflows/main.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        dotnet: ['net6.0', 'net472', 'net471']
+        dotnet: ['net5.0', 'net6.0', 'net7.0', 'net8.0', 'net471', 'net472', 'net481']
```

Comment:
In line 44 there is:
 ```
        uses: actions/setup-dotnet@v3
        with:
          dotnet-version: '6.0.x'
```
so we are using .NET 6 to build and run tests for .NET version 8. It looks suspicious to me. I would expect to have dotnet 8 version installed to test against dotnet 8.
If you look at .NET 8 build time executions they are only 2 or 3 minutes instead of 13 minutes. This is showing that the test weren't executed and we have false positive results.

Reply:
Good catch, the .net versions 5, 7, and 8 are only running the tests for 1 second while the .NET Framework versions are running for at least 10 minutes

I'll add the missing net versions

Reply:
Looks like the tests are running now after changing the `RuntimeFrameworkVersion` to 8